software/wedding-share.yml

```
@@ -0,0 +1,12 @@
+name: "WeddingShare"
+website_url: "https://github.com/Cirx08/WeddingShare"
+source_code_url: "https://github.com/Cirx08/WeddingShare"
+description: "Event photo sharing platform and gallery"
```

Could this be made a bit longer? And if not:
```suggestion
description: "Event photo sharing platform and gallery."
```